python/hsfs/storage_connector.py

```diff
@@ -720,6 +725,10 @@ def options(self):
         return self._options
 
     def snowflake_connector_options(self):
```

Comment:
is this method being used? otherwise can we remove it?

Reply:
it's a user-facing funciton, so I kept it for backwards compatability.